cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -851,18 +851,18 @@ Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
 #   CLUSTER_IP_RANGE
 #   SERVICE_CLUSTER_IP_RANGE
 function Configure-CniNetworking {
-  $CNI_RELEASE_VERSION = 'v0.8.0'
+  $CNI_RELEASE_VERSION = 'v0.8.2-gke.0'
```

Comment:
Why -gke ? Are these different than the plugins that are released on github?

Reply:
it is the same source, different build. i tested it manually, it works fine.